docs/product-user-guide/src/main/asciidoc/decision-tables-rule-table-entries-ref.adoc

```diff
@@ -0,0 +1,85 @@
+|At most one column.
+
+|`DESCRIPTION`
+|I
+|Results in a comment within the generated rule.
+|At most one column.
+
+|`CONDITION`
+|C
+|Code snippet and interpolated values for constructing a constraint within a pattern in a condition.
+|At least one per rule table.
+
+|`ACTION`
+|A
+|Code snippet and interpolated values for constructing an action for the consequence of the rule.
+|At least one per rule table.
+
+|`METADATA`
+|@
+|Code snippet and interpolated values for constructing a metadata entry for the rule.
+|Optional, any number of columns.
+|===
+
+// The cells that follow condition, action, or metadata attributes are used together to form the pattern for constraints, consequences, or annotations for the rules in the decision table.
+
+The following sections provide more details about how condition, action, and metadata columns use cell data:
+
+Conditions::
+
+For columns headed `CONDITION`, the cells in successive lines result in a conditional element:
+
+* *First cell:* Text in the first cell below `CONDITION` develops into a pattern for the rule condition, and uses the snippet in the next line as a constraint. If the cell is merged with one or more neighboring cells, a single pattern with multiple constraints is formed. All constraints are combined into a parenthesized list and appended to the text in this cell.
++
+If this cell is empty, the code snippet in the cell below it must result in a valid conditional element on its own. For example, instead of `Person` as the object type and `age` as the constraint (separate cells), you can enter `Person( age == $1)` in the object type cell and leave the constraint cell below it empty.
++
+To include a pattern without constraints, you can write the pattern in front of the text of another pattern, with or without an empty pair of parentheses. You can also append a `from` clause to the pattern.
++
+If the pattern ends with `eval`, code snippets produce boolean expressions for inclusion into a pair of parentheses after `eval`.
+* *Second cell:* Text in the second cell below `CONDITION` is processed as a constraint on the object reference in the first cell. The code snippet in this cell is modified by interpolating values from cells farther down in the column. If you want to create a constraint consisting of a comparison using `==` with the value from the cells below, then the field selector alone is sufficient. Any other comparison operator must be specified as the last item within the snippet, and the value from the cells below is appended. For all other constraint forms, you must mark the position for including the contents of a cell with the symbol `$param`. Multiple insertions are possible if you use the symbols `$1`, `$2`, etc., and a comma-separated list of values in the cells below. However, do not separate `$1`, `$2`, etc., by commas, or the table will fail to process.
```

Comment:
"interpolating" --- REALLY? Who knows this word?  :)

Reply:
Ha! It's one of those standard words when dealing with decision table logic (decision tables are a common thing in general for coding decision logic). And I guess I just got indoctrinated by all the upstream documentation. I'll think on it. Thanks.

Reply:
@sterobin @michelehaglund Wait until you write/look at the documents relating to decision table validation and verification... this uses the word "subsumption" -- and community documents may even mention "subsumptive" (there's a "made up" word if ever I read one!) ;-)